database/seeders/StandTemplateSeeder.php

```diff
@@ -44,18 +44,23 @@ public function run()
             DB::table(Stand::TABLE)->insert($stand);
         }
 
+        $const_shedule = [
```

Comment:
why did you name it `conts`? It is var 
```suggestion
        $week_shedule = [
```